modules/ROOT/pages/advanced/security.adoc

```diff
@@ -89,22 +94,32 @@ ____
 
```

Comment:
I can't comment on the exact list again sorry, but the admonition on line 76-93 has conversion issues. It should be a block admonition by the looks, like so:
```
[IMPORTANT]
====
These directives will prevent all content loading from external sources.
To allow content from specific sources, add the source domains to the relevant directives. For example, to allow YouTube videos:

[source, html]
----
media-src 'self' *.youtube.com;
----

To allow content from any source, use the (*) wildcard. Allowing all sources (using *) negates the security policy for the source type. For example:

[source, html]
----
media-src *;
----

For information on Content Security Policies, see: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP[MDN Web Docs - Content Security Policy (CSP)].
====
```